### ISSUE TITLE
Vaadin Repository Deployment Incomplete

### ISSUE THREAD

**Issue by mbruch** [mpilone/uitask-vaadin#1]
I tries to install this addon but maven failed to download it. It, however, could download sources and javadocs. The error message is:
Failed to execute goal on project XYZ: Could not resolve dependencies for project XYZ:XYZ:jar:2.0.0-SNAPSHOT: Failed to collect dependencies at org.mpilone.vaadin:uitask-vaadin:jar:1.0.0: Failed to read artifact descriptor for org.mpilone.vaadin:uitask-vaadin:jar:1.0.0: Failure to find org.mpilone.vaadin:uitask-vaadin-parent:pom:1.0.0 in http://maven.vaadin.com/vaadin-addons was cached in the local repository, resolution will not be reattempted until the update interval of vaadin-addons has elapsed or updates are forced -> [Help 1]

Is the deployment in the Vaadin addon repo incomplete?

```
8 -rw-r--r--   1 marcel  staff   267B Jul 11 12:00 _remote.repositories
8 -rw-r--r--   1 marcel  staff   529B Jul 11 12:00 m2e-lastUpdated.properties
112 -rw-r--r--   1 marcel  staff    52K Jul 11 12:00 uitask-vaadin-1.0.0-javadoc.jar
8 -rw-r--r--   1 marcel  staff    40B Jul 11 12:00 uitask-vaadin-1.0.0-javadoc.jar.sha1
16 -rw-r--r--   1 marcel  staff   5.6K Jul 11 12:00 uitask-vaadin-1.0.0-sources.jar
8 -rw-r--r--   1 marcel  staff    40B Jul 11 12:00 uitask-vaadin-1.0.0-sources.jar.sha1
16 -rw-r--r--   1 marcel  staff   6.3K Jul 11 12:00 uitask-vaadin-1.0.0.pom
8 -rw-r--r--   1 marcel  staff    40B Jul 11 12:00 uitask-vaadin-1.0.0.pom.sha1
```

**Comment by mpilone** [comment]
I missed the fact that the Vaadin directory doesn't support multi-module builds with Maven so the module can't refer to the parent POM. I'll get a new build published shortly.

More info here: https://vaadin.com/web/matti/blog/-/blogs/preparing-your-vaadin-add-on-for-dependency-management

**Comment by mpilone** [comment]
Fixed with ff77d3cf5fb8dbbb558c76c80c624d7904e78d6d in v1.1.0.
